<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormMenuHSet</class>
 <widget class="QWidget" name="FormMenuHSet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FormMenuHSet</string>
  </property>
  <property name="styleSheet">
   <string notr="true">FormMenuHSet {
background: gray;
}</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>50</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>128</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>系统参数</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>128</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>马达模式</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>128</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>自定马达模式</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_4">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>128</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>频率增益</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_5">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>128</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>选别秤</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_6">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>128</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>参数备份</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_7">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>128</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>模块状态</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QStackedWidget" name="page">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>1024</width>
     <height>590</height>
    </rect>
   </property>
   <widget class="FormMenuHSetSysParam" name="page_SysParam"/>
   <widget class="FormMenuHSetMM" name="page_MM"/>
   <widget class="FormMenuHSetMMA" name="page_MMA"/>
   <widget class="FormMenuHSetPL" name="page_PL"/>
   <widget class="FormMenuHSetSortor" name="page_Sortor"/>
   <widget class="FormMenuHSetPBak" name="page_PBak"/>
   <widget class="FormMenuHSetMainboard" name="page_Mainboard"/>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FormMenuHSetSysParam</class>
   <extends>QWidget</extends>
   <header>FormMenuHSetSysParam.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FormMenuHSetMM</class>
   <extends>QWidget</extends>
   <header>FormMenuHSetMM.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FormMenuHSetPBak</class>
   <extends>QWidget</extends>
   <header>FormMenuHSetPBak.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FormMenuHSetMainboard</class>
   <extends>QWidget</extends>
   <header>FormMenuHSetMainboard.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FormMenuHSetPL</class>
   <extends>QWidget</extends>
   <header>FormMenuHSetPL.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FormMenuHSetMMA</class>
   <extends>QWidget</extends>
   <header>FormMenuHSetMMA.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FormMenuHSetSortor</class>
   <extends>QWidget</extends>
   <header>FormMenuHSetSortor.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
